### PR TITLE
[PR] layered: #690 Extended documentation of 'positionChoiceConstraint'

### PR DESCRIPTION
**Pull request by uruuru**

### CODE REVIEW

**Review by soerendomroes — APPROVED** [review]
Looks good

**Comment by uruuru** [comment]
Good point. Thanks.

File: plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk [hunk 1]
```diff
@@ -228,9 +228,13 @@ group layering {
...
-                     Let i be the value of teh constraint. Assumed the drawing has n layers and i < n.
-                     If set to i, it expresses that the node should be placed in i-th layer.
-                     Should i>=n be true then the node is placed in the last layer of the drawing."
+             Let i be the value of teh constraint. Assumed the drawing has n layers and i < n.
+             If set to i, it expresses that the node should be placed in i-th layer.
+             Should i>=n be true then the node is placed in the last layer of the drawing.
+                     
+             Note that this option is not part of any of ELK Layered's default configurations 
+             but is only evaluated as part of the `InteractiveLayeredGraphVisitor`,
+             which must be applied manually or used via the `DiagramLayoutEngine."
```
Comment (le-cds) [inline plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk:237]: Missing closing `

File: plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk [hunk 2]
```diff
@@ -228,9 +228,13 @@ group layering {
         label "Layer Choice Constraint"
         description
             "Allows to set a constraint regarding the layer placement of a node. 
-                     Let i be the value of teh constraint. Assumed the drawing has n layers and i < n.
-                     If set to i, it expresses that the node should be placed in i-th layer.
-                     Should i>=n be true then the node is placed in the last layer of the drawing."
+             Let i be the value of teh constraint. Assumed the drawing has n layers and i < n.
```
Comment (le-cds) [inline plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk:231]: the

File: plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk [hunk 3]
```diff
@@ -392,10 +396,13 @@ group crossingMinimization {
...
-                     If set to i, it expresses that the node should be placed at the i-th position.
-                     Should i>=n be true then the node is placed at the last position in the layer.
-                    "
+             Assumed the layer in which the node placed includes n other nodes and i < n.
+             If set to i, it expresses that the node should be placed at the i-th position.
+             Should i>=n be true then the node is placed at the last position in the layer.
+             
+             Note that this option is not part of any of ELK Layered's default configurations 
+             but is only evaluated as part of the `InteractiveLayeredGraphVisitor`,
+             which must be applied manually or used via the `DiagramLayoutEngine. "
```
Comment (le-cds) [inline plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk:405]: Missing `

**Pull request merged**